novawallet/Modules/Staking/StakingSelectPool/Model/StakingSelectPoolViewModelFactory.swift

```diff
@@ -0,0 +1,77 @@
+import SoraFoundation
+
+protocol StakingSelectPoolViewModelFactoryProtocol: StakingSelectPoolViewModelFactory {
+    func createStakingSelectPoolViewModels(
```

Comment:
Let't move this to the existension. We don't really need to force implementation of this because it can be implemented based on single viewModel method.